Assets/js/resultspage.js

```diff
@@ -0,0 +1,169 @@
+$(document).ready(function () {
+  function yelpRequest(zipCode) {
+    foodRequest(zipCode);
+    eventRequest(zipCode);
+    $(".modal").modal();
+  }
+  // Gets the zip code from the URL
+  const urlParams = new URLSearchParams(window.location.search);
+  const zipCode = urlParams.get("zip");
+  yelpRequest(zipCode);
+});
+
+// Function that gets Yelp API request & builds UI content
+function foodRequest(zipCode) {
+  let yelpAPIKey =
```

Comment:
Ideally, you'll never put your API key in you GitHub repository.
Node has a great module called dot-env that can keep these secret for you in a `.env` file.